packages/components/src/components/as-histogram-widget/utils/draw.service.ts

```diff
@@ -139,7 +159,13 @@ export function renderYAxis(
     .tickSize(-WIDTH)
     .ticks(5)
     .tickPadding(10)
-    .tickFormat(format('.2~s'));
+    .tickFormat((value) => {
+      if (value > 0 && value < 1) {
```

Comment:
This can be abstracted to a format function and called in the two places it appears.

Reply:
You're absolutely right.